locations/container/pom.xml

```diff
@@ -32,10 +32,50 @@
     </parent>
 
     <properties>
-        <kubernetes-client.version>1.4.27</kubernetes-client.version>
+        <kubernetes-client.version>2.5.6</kubernetes-client.version>
     </properties>
 
     <dependencies>
+        <dependency>
+            <groupId>org.microbean</groupId>
+            <artifactId>microbean-helm</artifactId>
+            <version>2.5.0.1-SNAPSHOT</version>
+            <exclusions>
+                <!--<exclusion>-->
+                    <!--<groupId>io.grpc</groupId>-->
+                    <!--<artifactId>grpc-netty</artifactId>-->
+                <!--</exclusion>-->
```

Comment:
Can be removed

Reply:
thanks @tbouron 
also now there is an official release http://search.maven.org/#search%7Cga%7C1%7Ca%3Amicrobean-helm which may be better and needs testing

Reply:
Ah ok, I'll let you decide which one is best then ;)